cmd/wire/main.go

```diff
@@ -39,6 +39,7 @@ const usage = "usage: wire [gen|diff|show|check] [...]"
 
 func main() {
 	var err error
+	exitCode := 0
```

Comment:
nit: It would be nice for these variables to be introduced in the same order as they are assigned. I was thinking:

```go
var (
  exitCode int = 0
  err      error
)
```

Reply:
Done.